### PR TITLE
Setup & Make First Endpoint

### PR DESCRIPTION
**Pull request by kelshadarby**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling (Styling to new or existing files)
- [X] Testing
### Detailed Description
- Created a test for a dummy endpoint at '/'
- Created a dummy endpoint at '/' and made sure tests passed
- Set up a base flask app
### Why is this change required? What problem does it solve?
- We got the app set up and running to get things moving for BE
- We got a base API endpoint set up to make sure the app was in working order
### Was any previously written code changed?
- No
### Directory modified:
- [X] insert file names here
- At this point all files in app are new and will be merged in with this PR

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+__pycache__*
```
Comment (mikez321) [inline .gitignore:1]: Might need to do some research on why this is still not catching all those files.

File: instance/config.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import os
+
```
Comment (mikez321) [inline instance/config.py:2]: Not sure if we really need all of these but doesn't hurt I guess

File: test_endpoint.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+class EndpointTestCase(unittest.TestCase):
+    """This class represents the test case"""
+
+    def setUp(self):
+        """Define test variables and initialize app."""
+        self.app = create_app(config_name="testing")
+        self.client = self.app.test_client
+
+    def test_api_can_get_a_dummy_endpoint(self):
```
Comment (mikez321) [inline test_endpoint.py:14]: Yay! I'm glad this dummy endpoint works!

**Comment by mikez321** [comment]
Looks like a good start, this thing is a little more of a struggle than I was expecting!

File: .gitignore (hunk 1 above)
Comment (kelshadarby) [reply .gitignore:1]: I'll look into this a bit more this evening to try and figure out whats happening

**Pull request merged**